### PR TITLE
First of many steps of repository cleanup

### PR DESCRIPTION
**Pull request by Rouiller**

### CODE REVIEW

File: steno/src/model/stream/stream.cpp [hunk 1]
```diff
@@ -39,23 +39,11 @@ Stream::Stream(const VideoConfig& videoInputConfig, const VideoConfig& videoOutp
...
-    }
-
-    std::string configFile = root + "/config/yolo/cfg/yolov3-tiny.cfg";
-    std::string weightsFile = root + "/config/yolo/weights/yolov3-tiny.weights";
-    std::string metaFile = root + "/config/yolo/cfg/coco.data";
+    std::string configFile =
+        (QCoreApplication::applicationDirPath() + "/../configs/yolo/cfg/yolov3-tiny.cfg").toStdString();
+    std::string weightsFile =
+        (QCoreApplication::applicationDirPath() + "/../configs/yolo/weights/yolov3-tiny.weights").toStdString();
+    std::string metaFile = (QCoreApplication::applicationDirPath() + "/../configs/yolo/cfg/coco.data").toStdString();
```
Comment (alexmo16) [inline steno/src/model/stream/stream.cpp]: thanks god

Comment (Rouiller) [reply steno/src/model/stream/stream.cpp]: LOL

File: .gitignore [hunk 2]
```diff
@@ -1,23 +1,66 @@
...
+# QtCreator local machine specific files for imported projects
+*creator.user*
+
+# Custom
 .vscode/
 test_separated.raw
 test_postfiltered.raw
-scripts/ConfigurationHelper/ODASOutput.json
-*.pyc
 .coverage
```
Comment (alexmo16) [inline .gitignore]: you can remove .coverage

File: steno/3rd/v4l2/include/V4l2Device.h [hunk 3]
```diff
@@ -97,6 +100,9 @@ class V4l2Device
 		unsigned int m_format;
 		unsigned int m_width;
 		unsigned int m_height;	
+
+		struct v4l2_buffer m_partialWriteBuf;
+		bool m_partialWriteInProgress;
```
[the remark this answers is not in the document]
Comment (Rouiller) [reply steno/3rd/v4l2/include/V4l2Device.h]: The header file comes from the source repo. And anyway, I can't tell why git compared those 2 files.

File: .travis.yml [hunk 4]
```diff
@@ -1,19 +1,14 @@
...
 cache: 
-  - pip
   - ccache
 compiler: 
   - g++
 before_install:
   - sudo apt-get update
 install:
-  - pip install -r ./requirements.txt || travis_terminate 1
   - sudo apt-get install -y xorg-dev libglu1-mesa-dev libpulse-dev || travis_terminate 1
```
[the remark this answers is not in the document]
Comment (Rouiller) [reply .travis.yml]: I removed xorg, but libglu1 is required

**Pull request merged**